util.js

```diff
@@ -33,7 +34,15 @@ exports.getAvaConfig = filepath => {
 
 	try {
 		const packageContent = JSON.parse(fs.readFileSync(filepath, 'utf8'));
-		return (packageContent && packageContent.ava) || defaultResult;
+
+		if (packageContent && packageContent.ava) {
```

Comment:
Strictly speaking AVA ignores `"ava": {}` in favor of an `ava.config.js` file. We should do the same here.

Reply:
From the docs:

```
To use an ava.config.js file:

It must be in the same directory as your package.json
Your package.json must not contain an ava property (or, if it does, it must be an empty object)
```
I thought it would prefer the ava object from package.json

Reply:
No it throws an exception if it finds config in `package.json` (that is, the `"ava"` key contains a non-empty object) as well as an `ava.config.js` file. Therefore if you find an empty object here, you need to check for `ava.config.js` as well. But it may not exist, and that should be OK.

I'm not sure what we should do here when both configurations are provided. If we throw an exception how does ESLint surface that?

Reply:
When you throw an error in ESLint, it just ends the process. So I think throwing an error isn't an option here.